src/components/CirclePagination/CirclePagination.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import { Row, Box } from "../../theme/custom-styled-components";
+import { ColorType } from "../../theme/util";
+import Circle from "./Circle";
+
+export interface CirclePaginationProps {
+  activeItem: number;
+  data: any;
+  bgColor?: ColorType;
+  onItemClick: (index: any) => void;
+  className?: string;
+}
+
+const ButtonStyled = styled(Box)`
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+`;
+
+export const CirclePagination: FC<CirclePaginationProps> = ({
+  onItemClick,
+  data,
+  bgColor,
+  activeItem,
+  className,
+}: CirclePaginationProps) => {
+  return (
+    <Row className={className} mt={{ xxs: 3, lg: 5 }}>
+      {data.map((item: any, index: number) => {
+        return (
+          <ButtonStyled
+            aria-label={`carousel-page-${index}`}
+            as="button"
+            key={`pagination-element-${index}-${item.toString()}`}
+            mr={{ xxs: 5, lg: 7 }}
```

Comment:
We could make the component more robust by using CSS Grid and `grid-gap` property instead of applying margin right here because there could be a use-case where we want to centre the pagination and not have a right margin on the last item.